quality/metrics/bertscore.md

```diff
@@ -0,0 +1,19 @@
+---
+grand_parent: Building and research
+parent: Metrics
+nav_order: 7
+title: BERTScore
+description:
+---
+
+The **BERTScore** metric calculates the similarity in meaning between the machine translation output and a reference.
+BERTScore uses contextual embeddings.
+
+> BERTScore addresses two common pitfalls in n-gram-based metrics. [...] First, such methods often fail to robustly match paraphrases. [...] Second, n-gram models fail to capture distant dependencies and penalize semantically-critical ordering changes For example, given a small window of size two, BLEU will only mildly penalize swapping of cause and effect clauses (e.g. A because B instead of B because A), especially when the arguments A and B are long phrases. In contrast, contextualized embeddings are trained to effectively capture distant dependencies and ordering.
```

Comment:
Let's add an empty line between each sentence in such quotes, even though it wasn't in the original.